pkg/nodegraph/Library/NodegraphRecipe.js

```diff
@@ -97,16 +107,19 @@ export const NodegraphRecipe = {
       },
       editor: {
         Editor: {
-          $kind: '$library/NodeGraph/Editor',
+          //$kind: 'https://rapsai-core.web.app/0.5.1/Library/Editor',
+          $kind: 'http://localhost:9876/Library/Editor',
```

Comment:
can it be a relative path? for example, i'm not running the local server at port 9876, so this will be failing by default

Reply:
You can put whatever you want there. You are right that the checked-in path is for debugging my machine, the commented out one is the correct general purpose URL.